scripts/assets/tests/test-os.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -ex
```

Comment:
I think you can remove `-x` here (maybe a debugging leftover?)

Reply:
Updated